src/webviews/webview-side/interactive-common/variableExplorerGrid.less

```diff
@@ -40,7 +40,7 @@
     padding: 2px;
     font-family: var(--vscode-font-family);
     border-right: 1px solid var(--vscode-menu-border);
-    padding-top: 4px;
+    padding: 4px 8px;
```

Comment:
Can we do the same here, just add padding-top and padding-bottom

Reply:
Oh, my bad! I had forgotten I changed it in two places. On my way!

Reply:
Sorry, here it doesn't work!

Reply:
Oh I see why. Disregard!

Reply:
Fixed here: https://github.com/microsoft/vscode-jupyter/pull/10747/commits/0fb043be585c47f8aa30fc2bb7b69eb968b3c9ce